src/platform_impl/macos/window.rs

```diff
@@ -1071,6 +1071,21 @@ impl UnownedWindow {
     }
   }
 
+
+  #[inline]
+  pub fn is_document_edited(&self) -> bool {
+    unsafe {
+      self.ns_window.isDocumentEdited()
+    }
+  }
+
+  #[inline]
+  pub fn set_is_document_edited(&self, edited: bool) {
+    unsafe {
+      self.ns_window.setDocumentEdited_(edited);
+    }
+  }
+
```

Comment:
no need to have double implementation, the below implementation is enough

Reply:
Thanks, I think I got lost there. Read through the types and it makes more sense now.